recipes/msys2/all/conanfile.py

```diff
@@ -11,19 +12,23 @@ class MSYS2Conan(ConanFile):
     homepage = "http://www.msys2.org"
     license = "MSYS license"
     topics = ("conan", "msys", "unix", "subsystem")
-    build_requires = "7zip/19.00"
     short_paths = True
     options = {"exclude_files": "ANY",  # Comma separated list of file patterns to exclude from the package
                "packages": "ANY",  # Comma separated
                "additional_packages": "ANY"}    # Comma separated
     default_options = {"exclude_files": "*/link.exe",
                        "packages": "base-devel,binutils,gcc",
                        "additional_packages": None}
-    settings = "os_build", "arch_build"
+    settings = "os", "arch"
 
-    def configure(self):
-        if self.settings.os_build != "Windows":
+    def validate(self):
+        if not tools.os_info.is_windows:
```

Comment:
that's probably the line discarding all the packages, because CI initially tries to export recipe on Linux to calculate all possible package IDs